vendor/the-things-products/the-things-uno-codec.yaml

```diff
@@ -0,0 +1,53 @@
+uplinkDecoder:
+  fileName: the-things-uno.js
```

Comment:
The problem with this device is that people can customize the firmware. There are quite some (examples)[https://github.com/TheThingsNetwork/arduino-device-lib/tree/master/examples] that all have different payload codecs.

I don't think we can declare a codec for The Things Uno.

Reply:
can we declare that it matches the Quick Start arduino example?

Reply:
Yes, but then I would declare that as a firmware version. So different sketches get a different firmware version; sketches are basically just that.